game/plugin/navigation/door/meta.toml

```diff
@@ -0,0 +1,7 @@
+name = "Door"
```

Comment:
We aren't using `meta.toml` anymore, can you delete this file and add the appropriate information to `build.gradle` instead please (you can look at any of the existing plugins for the format).

Also, *please* add the authors of the previous door plugin to the authors list.